### PR TITLE
Replaced jQuery DOM Selectors with in-built one

### LINKED ISSUE #111: Replace use of jQuery add/remove class
We are using jQuery DOM selectors to toggle classes and visibility in main.js but now all of those functionalities are part of the browser built-in functions.

For all usages of CSS class add and remove functions, e.g.

https://github.com/Techtonica/keyboard-shortcuts-practice/blob/27c54db1ce3187f2f40434fdabe677976d53cedc/public/scripts/main.js#L116

or 

https://github.com/Techtonica/keyboard-shortcuts-practice/blob/27c54db1ce3187f2f40434fdabe677976d53cedc/public/scripts/main.js#L131

Replace with the equivalent using the standard `document.querySelector` and [`Element.classList`](https://developer.mozilla.org/en-US/docs/Web/API/Element/classList)

### PR DESCRIPTION
**Pull request by i-aryan**
Fixes #111

Replaced jQuery DOM Selectors with in-built one where addClass or removeClass was used.

### CODE REVIEW

File: public/scripts/main.js [hunk 1]
```diff
@@ -253,7 +253,7 @@ function updateTimingDisplay() {
 
 function onIncorrect() {
   $('#textdiv').effect("shake", { distance: 3 });
-  $("#read").addClass('incorrect');
+  document.querySelector("#read").classList.add('incorrect');
   setTimeout(clearPressedKeys, 500);
 };
```
Comment (apoorva-mk) [inline public/scripts/main.js:259]: Please resolve the conflicts and also if possible reference the issue in the header.

Comment (i-aryan) [reply public/scripts/main.js:303]: @apoorva-mk Done!

**Review by vegetabill — CHANGES_REQUESTED** [review]
When I tried to test this in Chrome, I got a few JavaScript errors in the console and it no longer advanced to the next question.

```
main.js:273 Uncaught TypeError: Cannot read property 'classList' of null
    at clearPressedKeys (main.js:273)
    at main.js:413
```

(I cleared my local storage to start again at the first question and it still happened)

Can you check your console and see if there are any errors? One difference with the new code and jQuery is if you use a selector with no matched elements, it lets you call any functions on it and just doesn't do anything. Sorry if that wasn't clear.

**Comment by i-aryan** [comment]
@vegetabill Hey, I've tried to make changes again. If it doesn't work this time please mark the PR invalid and I'll open a new one by forking the latest repository.

**Review by vegetabill — APPROVED** [review]
Looks good! Thanks for revising it.

**Pull request merged**